src/services/auth-services.ts

```diff
@@ -7,23 +7,6 @@ import { StatusCodes } from 'http-status-codes'
  * Class that provides integration with the Auth API.
  */
 export default class AuthServices {
-  /**
-   * Fetches authorizations.
-   * @param businessIdentifier the business identifier (eg, BC1219948)
-   * @returns a promise to return the authorizations object
-   */
-  static async fetchAuthorizations (businessIdentifier: string): Promise<any> {
-    if (!businessIdentifier) throw new Error('Invalid parameter \'businessIdentifier\'')
-
-    const authApiUrl = sessionStorage.getItem('AUTH_API_URL')
-    const url = `${authApiUrl}entities/${businessIdentifier}/authorizations`
-
-    return axios.get(url).then(response => {
-      if (response?.data) return response.data
-      throw new Error('Invalid response data ')
-    })
-  }
```

Comment:
We no longer need this. In a future ticket, it will be replaced with a similar call to Legal API.